test/autest/gold_tests/static_file/static_file.test.py

```diff
@@ -12,7 +12,10 @@
 '''
 r = Test.TxnBoxTestRun("Test static file support", "static_file.replay.yaml", config_key="meta.txn_box")
 ts = r.Variables.TS
+server = r.Variables.SERVER
 ts.Setup.Copy("static_file.txt", ts.Variables.CONFIGDIR)
 ts.Disk.remap_config.AddLine(
-    'map http://example.one http://example.one @plugin=txn_box.so @pparam=static_file.replay.yaml @pparam=meta.txn-box-remap'
+        'map http://example.one http://127.0.0.1:{} '
```

Comment:
I was looking at this issue, but that's the wrong solution. The problem is it modifies the client request URL and `Host` field, which breaks tests that depend on those. Because micro-DNS is now enabled, this should remap to "example.one:{}".

Reply:
Oh, right.

This is addressed by your TxnBoxTestRun remap parameter suggestion.